vendor/vendor.json

```diff
@@ -273,6 +273,14 @@
 			"revisionTime": "2018-11-28T19:21:57Z",
 			"tree": true
 		},
+		{
+			"checksumSHA1": "Wu1rZtI1n4bDQPyLuK60cDFmvFI=",
+			"path": "github.com/keep-network/blake2b/compression",
+			"revision": "02884c605a94951d05165fd56c69d6239f63b3d1",
+			"revisionTime": "2019-07-02T09:51:07Z",
+			"version": "v1.0",
+			"versionExact": "v1.0"
```

Comment:
Since this is actually code taken verbatim from Go (as far as I understand it), please move it to `crypto/blake2b` inside the repo. I don't like the idea that we have a middleman repo in between for code that's copy pasted from elsewhere anyway.

Reply:
This is not entirely copy-paste. Go code does not expose `F` compression function, we had to modify it to allow for an arbitrary number of rounds, have the interface compatible with what we expect from the precompile and removed some pieces that are not really a part of compression but rather hashing. Please compare:
https://github.com/keep-network/blake2b/blob/master/compression/f.go#L39
https://github.com/golang/crypto/blob/master/blake2b/blake2b_generic.go#L30